k8s/file/provider_test.go

```diff
@@ -20,3 +27,65 @@ func TestGetObjectsNamespaceError(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func Test_provider_GetObjects(t *testing.T) {
```

Comment:
don't snake case functions

Reply:
That's an autogenerated test skeleton using gotests

Reply:
That's an autogenerated test skeleton using gotests